snuba/datasets/storages/spans.py

```diff
@@ -0,0 +1,90 @@
+from snuba.clickhouse.columns import (
+    UUID,
+    Array,
+    ColumnSet,
+    DateTime,
+    LowCardinality,
+    Materialized,
+    Nested,
+    String,
+    UInt,
+    WithDefault,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas import MandatoryCondition
+from snuba.datasets.schemas.tables import ReplacingMergeTreeSchema
+from snuba.datasets.spans_processor import UNKNOWN_SPAN_STATUS, SpansMessageProcessor
+from snuba.datasets.storage import WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.tags_hash_map import TAGS_HASH_MAP_COLUMN
+from snuba.datasets.table_storage import KafkaStreamLoader
+from snuba.query.conditions import ConditionFunctions, binary_condition
+from snuba.query.expressions import Column, Literal
+from snuba.query.processors.prewhere import PrewhereProcessor
+from snuba.web.split import TimeSplitQueryStrategy
+
+columns = ColumnSet(
+    [
+        ("project_id", UInt(64)),
+        ("transaction_id", UUID()),  # event_id of the transaction
+        ("span_id", UInt(64)),
+        ("transaction_name", LowCardinality(String())),
+        ("parent_span_id", UInt(64)),  # This should be able to replace transaction_id
```

Comment:
Again I think this might need to be Nullable. Or we should enforce that the root span's parent is itself or something.

Reply:
see comment above